spec/system/prior_authority/application_lists_spec.rb

```diff
@@ -3,10 +3,10 @@
 RSpec.describe 'Prior authority application lists' do
   before do
     visit provider_saml_omniauth_callback_path
-    create(:prior_authority_application, laa_reference: 'LAA-AAAAA', status: 'submitted', provider: Provider.first)
-    create(:prior_authority_application, laa_reference: 'LAA-BBBBB', status: 'submitted', provider: Provider.first)
-    create(:prior_authority_application, laa_reference: 'LAA-CCCCC', status: 'granted', provider: Provider.first)
-    create(:prior_authority_application, laa_reference: 'LAA-DDDDD', status: 'draft', provider: Provider.first)
+    create(:prior_authority_application, laa_reference: 'LAA-AAAAA', status: 'submitted', created_at: 1.day.ago)
+    create(:prior_authority_application, laa_reference: 'LAA-BBBBB', status: 'submitted', created_at: 2.days.ago)
+    create(:prior_authority_application, laa_reference: 'LAA-CCCCC', status: 'granted', created_at: 3.days.ago)
+    create(:prior_authority_application, laa_reference: 'LAA-DDDDD', status: 'draft', created_at: 4.days.ago)
```

Comment:
so we should have a test that only applications for the current office are shown in the list - I have been getting errors when trying to access applicartion but it fails due to office code filtering later in the process. :)

Reply:
Yeah... it's because to this point we've been ignoring office codes and simply looking at what applications the provider directly owns... I'll go ahead and fix that...